pkg/landscaper/controllers/installations/reconcile.go

```diff
@@ -158,6 +158,11 @@ func (c *Controller) handleReconcilePhase(ctx context.Context, inst *lsv1alpha1.
 func (c *Controller) handlePhaseInit(ctx context.Context, inst *lsv1alpha1.Installation) (lserrors.LsError, lserrors.LsError) {
 	currentOperation := "handlePhaseInit"
 
+	err := c.checkForDuplicateExports(ctx, inst)
+	if err != nil {
+		return nil, lserrors.BuildLsError(err, currentOperation, "CheckForDuplicateExports", err.Error(), lsv1alpha1.ErrorConfigurationProblem)
```

Comment:
```suggestion
		return lserrors.BuildLsError(err, currentOperation, "CheckForDuplicateExports", err.Error(), lsv1alpha1.ErrorConfigurationProblem), nil
```

Reply:
I would return this as a fatal error instead because manual intervention is required in that case.